doc-content/enterprise-only/getting-started-kogito/proc-kogito-migrate-external-application-rest.adoc

```diff
@@ -0,0 +1,15 @@
+[id="proc-kogito-migrate-external-application-REST_{context}"]
+= Migrating external applications to DMN model specific REST endpoints
+
+[role="_abstract"]
+To migrate a DMN project to {KOGITO} deployment, you first need to migrate external applications using the REST endpoints that are specific to DMN models. The external applications interact with the decision services on the {KIE_SERVER}.
```

Comment:
as above:

```suggestion
To migrate a DMN project to {KOGITO} deployment, you first should migrate external applications that are interacting with decision services on the {KIE_SERVER}, to use {KIE_SERVER} REST endpoints that are specific to DMN models. The external applications interact with the decision services on the {KIE_SERVER}.
```

Reply:
Similarly, changing this paragraph as follows:

To migrate a DMN project to {KOGITO} deployment, you first must migrate external applications that are interacting with decision services on the {KIE_SERVER}, using {KIE_SERVER} REST endpoints that are specific to DMN models.